bin/storm.py

```diff
@@ -94,7 +94,7 @@ def init_storm_env():
 CONFFILE = ""
 JAR_JVM_OPTS = shlex.split(os.getenv('STORM_JAR_JVM_OPTS', ''))
 JAVA_HOME = os.getenv('JAVA_HOME', None)
-JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java')
+JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java' if not is_windows() else "java.exe")
```

Comment:
This one-liner becomes a bit complicated now. Could we expand this to multiple lines? 
And isn't the difference between non-windows and windows be 'java' or 'java.exe'? I mean `java vs java.exe` or `<JAVA_HOME>/bin/java vs <JAVA_HOME>\bin\java.exe` might be correct whereas this line doesn't do the thing. Please let me know if I'm missing here.